### PR TITLE
PSG-5176

### PR DESCRIPTION
**Pull request by SinaSeylani**
Options added to register method

### CODE REVIEW

File: Sources/PassageFlex/PassagePasskeyAuthentication.swift [hunk 1]
```diff
@@ -21,7 +21,8 @@ internal struct PassagePasskeyAuthentication {
         let registrationRequest = try PasskeyRegistrationRequest.from(startResponse)
         let authController = PasskeyAuthorizationController()
         let credential = try await authController.requestPasskeyRegistration(
-            registrationRequest: registrationRequest
+            registrationRequest: registrationRequest,
+            includeSecurityKeyOption: authenticatorAttachment == .crossPlatform
```
Comment (rickycpadilla) [inline Sources/PassageFlex/PassagePasskeyAuthentication.swift:25]: Lets default to `.platform` here.
```suggestion
            includeSecurityKeyOption: authenticatorAttachment == .platform
```

**Review by rickycpadilla — COMMENTED** [review]
One minor change, otherwise looks great!

File: Sources/PassageFlex/PassagePasskeyAuthentication.swift (hunk 1 above)
Comment (SinaSeylani) [reply Sources/PassageFlex/PassagePasskeyAuthentication.swift:25]: This is not a default value. I'm setting the value of `includeSecurityKeyOption` as true if those two are equal

Comment (rickycpadilla) [reply Sources/PassageFlex/PassagePasskeyAuthentication.swift:25]: omg sorry, I misread 🤦‍♂️

**Review by rickycpadilla — APPROVED** [review]

**Pull request merged**